src/main/scala/mimir/Database.scala

```diff
@@ -521,8 +525,8 @@ case class Database(backend: Backend)
     val typeMap = schema.map { x => (x._1.toUpperCase -> x._2) }.toMap
     backend.getTableSchema(name) match {
       case None => {
-        val schemaElements = 
-          schema.map { case (name, t) => s"$name $t" } ++ 
+        val schemaElements =
+          schema.map { case (name, t) => s"$name $t" } ++
           (if(primaryKey.isEmpty) { Seq() } else {
             Seq(s"PRIMARY KEY (${primaryKey.get})")
           })
```

Comment:
Nearly all of the changes in this file (and most others) are whitespace changes.  
1. This makes reviewing far harder since there's numerous spurious edits.
2. This creates numerous additional opportunities for merge conflicts, which are annoying to deal with.

Before accepting this commit, we will need you to:
1. Please revert all changes to files that have no meaningful edits in them.  
2. Please revert all whitespace-only edits to files that do have meaningful edits.

Moving on, it would be helpful if you disabled the feature of your editor that automatically modifies whitespace throughout a file prior to editing files in this repository.